easybuild/easyblocks/c/cp2k.py

```diff
@@ -228,12 +228,30 @@ def configure_step(self):
             else:
                 options['DFLAGS'] += ' -D__ELPA3'
 
-        # CUDA
+        # CUDA support
+        # see https://github.com/cp2k/cp2k/blob/master/INSTALL.md#2j-cuda-optional-improved-performance-on-gpu-systems
         cuda = get_software_root('CUDA')
         if cuda:
-            options['DFLAGS'] += ' -D__ACC -D__DBCSR_ACC'
-            options['LIBS'] += ' -lcudart -lcublas -lcufft -lrt'
-            options['NVCC'] = ' nvcc'
+            # check GPU architecture(s) to build for
+            cuda_cc_sm_vals = self.cfg.template_values.get('cuda_sm_comma_sep')
+            if cuda_cc_sm_vals:
+                gpu_targets = '--gpu-code=%s' % cuda_cc_sm_vals
+            else:
+                gpu_targets = ''
```

Comment:
This needs a better default, and/or a warning if no CUDA compute capabilities were specified?